src/engine/ReluConstraint.cpp

```diff
@@ -829,6 +864,21 @@ unsigned ReluConstraint::getAux() const
     return _aux;
 }
 
+double ReluConstraint::computePolarity() const
+{
+    double currentLb = _lowerBounds[_b];
+    double currentUb = _upperBounds[_b];
+    if ( currentLb >= 0 || currentUb <= 0 ) return 1;
```

Comment:
why currentUb <= 0?

Reply:
is it `if ( currentLb >= 0 && currentUb <= 0 )` ?

Reply:
Whoops this is a bug. Fixed it and added more explanation in the header.